release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -557,6 +557,13 @@ For more information, see xref:../networking/k8s_nmstate/k8s-nmstate-updating-no
 
 {product-title} {product-version} now includes CoreDNS version 1.11.3.
 
+[id="ocp-4-17-expose-mtu"]
+==== Exposing MTU for vfio-pci SR-IOV devices
+
+With this release, MTU on virtual function using the vfio-pci driver is available in the network-status pod annotation, and inside the container. 
```

Comment:
Is MTU a very common term for this topic? If not, I would suggest spelling it out on the first mention

Reply:
Thanks @skopacz1 for your feedback. I have applied your suggestion and spelt it out.